Gulpfile.js

```diff
@@ -153,7 +154,7 @@ gulp.step('client-scripts-bundle', () => {
         // HACK: babel-plugin-transform-es2015-modules-commonjs forces
         // 'use strict' insertion. We need to remove it manually because
         // of https://github.com/DevExpress/testcafe/issues/258
-        return { code: transformed.code.replace(/^('|")use strict('|");?/, '') };
+        return { code: transformed.code.replace(/^(['"])use strict\1;?/, '') };
```

Comment:
This function is called for each client script file. So, it's better to mark it as constant at the start of the `Gulpfile.js`.